frontend/src/constansts/app.ts

```diff
@@ -0,0 +1,29 @@
+export enum Paths {
+    Profile = "/profile",
+    TopUp = "/topup",
+    Transactions = "/transactions",
+    Home = "/home",
+    SignIn = "/signin",
+    SignUp = "/signup"
+}
+
+export enum Titles {
+    APP_NAME = "Qpay",
+    GENERATION_QR = "QR Generation",
+    TRANSACTIONS = "Transactions",
+    HOME = "Home",
+    SIGN_IN = " Sign in",
+    SIGN_UP = "Sign Up",
+    PAYMENT = "Payment",
+    PROFILE = "Profile",
+    TOP_UP = "Top up",
+    LOGO = "Logo"
+}
+
+export enum Headers {
+    HOME = "Home",
+    TOP_UP = "Top Up",
+    PROFILE = "Profile",
+    TRANSACTIONS = "Transactions",
+    LOG_OUT = "Logout"
```

Comment:
```suggestion
    LOG_OUT = "Log out"
```
https://forums.developer.apple.com/forums/thread/6680#:~:text=Logout%20is%20a%20noun%2C%20to,you%20need%20to%20log%20out%22.